### PR TITLE
CVE-2011-1452 and CVE-2015-1275

### PR DESCRIPTION
**Pull request by Jcuevas97**
Pushed after the deadline CVE-2011-1452 and CVE-2015-1275

### CODE REVIEW

File: cves/CVE-2015-1275.yml [hunk 1]
```diff
@@ -1,200 +1,215 @@
...
+- :commit: 1d7da23e937a8c82a5d485c0b755b119ceb91ea1
+  :note: This was the creation of the URLUtilities file which was made for upstreaming WebAddress
+upvotes_instructions: |
+  For the first round, ignore this upvotes number.
+
+  For the second round of reviewing, you will be giving a certain amount of
+  upvotes to each vulnerability you see. Your peers will tell you how
+  interesting they think this vulnerability is, and you'll add that to the
+  upvotes score on your branch.
+upvotes:
```
Comment (andymeneely) [inline cves/CVE-2015-1275.yml]: +7 upvotes from me

File: cves/CVE-2011-1452.yml [hunk 2]
```diff
@@ -1,201 +1,238 @@
...
-    every one. Write a thoughtful entry here that those ing the software
-    engineering industry would find interesting.
-  answer: 
+---
+CVE: CVE-2011-1452
+CWE_instructions: |
+  Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
+  that describes your vulnerability. (Tip: this may not be a good one to start
+  with - spend time understanding this vulnerability before making your choice!)
+CWE:
```
Comment (blackpan2) [inline cves/CVE-2011-1452.yml]: CWE needs to be filled in

File: cves/CVE-2015-1275.yml [hunk 3]
```diff
@@ -1,200 +1,215 @@
...
-    every one. Write a thoughtful entry here that those ing the software
-    engineering industry would find interesting.
-  answer: 
+---
+CVE: CVE-2015-1275
+CWE_instructions: |
+  Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
+  that describes your vulnerability. (Tip: this may not be a good one to start
+  with - spend time understanding this vulnerability before making your choice!)
+CWE:
```
Comment (blackpan2) [inline cves/CVE-2015-1275.yml]: CWE needs to be filled in

File: cves/CVE-2011-1452.yml [hunk 4]
```diff
@@ -1,201 +1,238 @@
...
+    The "google" flag can be true, false, or nil.
+
+    If there is no evidence as to how this vulnerability was found, then you may
+    leave the entries blank except for "answer". Write down where you looked in "answer".
+  answer: |
+    I checked the Chromium bug report and it seems that it was found manually. The user
+    who found the vulnerability was able to construct various potential exploits of this
+    vulnerability before it was patched.
+  date: 2001-03-29
+  automated:
```
Comment (bencrossgrove) [inline cves/CVE-2011-1452.yml]: As it was found manually, set automated to false.

File: cves/CVE-2015-1275.yml [hunk 5]
```diff
@@ -1,200 +1,215 @@
...
+- 1059413004
+bugs:
+- 462843
+repo: 
+fixes_vcc_instructions: |
+  Please put the commit hash in "commit" below (see my example in
+  CVE-2011-3092.yml). Fixes and VCCs follow the same format.
+fixes:
+- :commit: e8b8c72eb8eb462c52a707c556d53d350bbd408d
+  :note: ''
```
Comment (bencrossgrove) [inline cves/CVE-2015-1275.yml]: Is there anything to note about this fix? Perhaps a commit message can be placed here.

**Pull request closed without merging**